packages/web-components/stencil.config.ts

```diff
@@ -1,14 +1,21 @@
 import { Config } from '@stencil/core';
 import { postcss } from '@stencil/postcss';
+import { sass } from '@stencil/sass';
 import url from 'postcss-url';
+import postcssSass from '@csstools/postcss-sass';
 import { reactOutputTarget } from '@stencil/react-output-target';
 
 export const config: Config = {
   namespace: 'component-library',
   globalStyle: 'src/global/main.css',
   plugins: [
+    sass({ includePaths: ["../../node_modules/@uswds/uswds/packages/"],
+      injectGlobalPaths: [
+        'src/global/settings.scss',
+      ]
+    }),
     postcss({
-      plugins: [url({ url: 'inline' })],
+      plugins: [postcssSass({ verbose: true, loadPaths: ["../../node_modules/@uswds/uswds/foo"] }), url({ url: 'inline' })],
```

Comment:
The [documentation for the JS api](https://sass-lang.com/documentation/js-api/interfaces/Options#loadPaths) shows the `loadPaths` option that is an array of strings, but I couldn't get this to work. I don't know where the bug is, but without this working I was getting errors like this:

![Build error on a css file with the message "File to import not found: `@import 'uswds-helpers/src/styles/_usa-sr-only`](https://user-images.githubusercontent.com/2008881/180918298-c16e3e62-d71f-4433-b458-4797ba76b674.png)

Within the CSS that we control I could change the import locations to be something like `../../node_modules/@uswds/uswds/packages/...`, but within USWDS sass files there are package imports that the compiler wasn't resolving. With the `@stencil/sass` plugin we can set `includePaths` and don't have to deal with these issues.